packages/launcher/src/index.tsx

```diff
@@ -218,27 +449,46 @@ export class Launcher extends VDomRenderer<LauncherModel> {
             </div>
           </div>
         );
+      }
+      if (categories[cat] && categories[cat].length > 0) {
         sections.push(section);
       }
     });
 
     // Wrap the sections in body and content divs.
     return (
-      <div className="jp-Launcher-body">
-        <div className="jp-Launcher-content">
-          <div className="jp-Launcher-cwd">
-            <h3>{this.cwd}</h3>
+      <React.Fragment>
+        <div className="jp-Launcher-body">
+          <div className="jp-Launcher-content">
+            <div className="jp-Launcher-toolbar">
+              <div className="jp-Launcher-search-div">
```

Comment:
The search and cwd UX should be moved to the actual activity toolbar - here we end up with a double horizontal line above and below this UX.